packages/httpsnippet-client-api/src/index.ts

```diff
@@ -164,7 +164,7 @@ const client: Client<APIOptions> = {
       }
     } else {
       sdkPackageName = getProjectPrefixFromRegistryUUID(opts.api.registryURI);
-      sdkVariable = 'sdk';
+      sdkVariable = camelCase(sdkPackageName as string);
```

Comment:
Probably don't want to suppress the TS error here that `sdkPackageName` might be `null` because that'll happen if `registryURI` isn't properly formulated.

```suggestion
      sdkVariable = camelCase(sdkPackageName || 'sdk');
```

Reply:
ah, good call!